app/views/support_interface/application_forms/application_choices/change_offered_course/change_offered_course_search.html.erb

```diff
@@ -15,9 +15,8 @@
       <h1 class="govuk-heading-l">Change offered course</h1>
 
       <ul class="govuk-list govuk-list--bullet">
-        <li>A candidate can only accept one of the courses offered to them.</li>
+        <li>This feature can only be used when a candidate has accepted the providers offer.</li>
```

Comment:
Is this line needed? Surely the link to change the offered course doesn't appear until this is the case?

Reply:
Yea i guess it can come out